Dockerfile.slim

```diff
@@ -40,9 +40,12 @@ COPY resources/ext/config/logback.xml /etc/puppetlabs/puppetdb/
 COPY resources/ext/config/request-logging.xml /etc/puppetlabs/puppetdb/
 RUN mkdir -p /opt/puppetlabs/server/data/puppetdb
 
+RUN addgroup $GROUP && adduser -S $USER -G $GROUP
+
+ADD https://raw.githubusercontent.com/puppetlabs/pupperware/6487f9f9bee24a5330b1709cefee8cc10c4bf6d2/shared/ssl.sh /ssl.sh
```

Comment:
Now that `ssl.sh` is merged, I think we can update this to a better URL

Reply:
Done: https://github.com/puppetlabs/puppetdb/pull/2775/commits/1829762d70210e52b5c9e2bdd8e1545a00e6bc60